Mac-App/Frontend/Frontend/Views/ExpandableList/ExpandableListView.swift

```diff
@@ -9,25 +9,22 @@
 import SwiftUI
 
 public struct ExpandableListView: View {
-    @ObservedObject private var viewData: ObservableData<DependencyTree.State>
+    @ObservedObject private var dependencyTreeState: Observable<DependencyTree.State>
```

Comment:
I had to touch this file due to the renaming. I'll open a separate PR to align the rest to the new naming.